### PR TITLE
Handle forward and backward compatibility

### PR DESCRIPTION
**Pull request by Aniket21mathur**
Src -> https://github.com/Aniket21mathur/ProtocolBuffers/blob/master/chapel/src/unknownfield.proto
Out -> https://github.com/Aniket21mathur/ProtocolBuffers/blob/master/chapel/build/unknownfield.chpl

### CODE REVIEW

File: src/plugin/message.cc [hunk 1]
```diff
@@ -69,6 +69,9 @@ namespace chapel {
       "}\n"
       "\n");
 
+    printer->Print("var unknowFieldStream: bytes = \"\";\n");
```
Comment (lydia-duncan) [inline src/plugin/message.cc]: Looks like you've got a typo here: missing the last "n" in "unknown"

Comment (lydia-duncan) [reply src/plugin/message.cc]: It happens at other parts of the PR as well, as a heads up

Comment (Aniket21mathur) [reply src/plugin/message.cc]: Ahhh :sweat_smile: , thanks for pointing out I will fix it!

**Comment by Aniket21mathur** [comment]
@lydia-duncan fixed the typo in cfd0110, it would be great if we can have a final review on this.

**Review by lydia-duncan — APPROVED** [review]
Looks good to me!

**Comment by Aniket21mathur** [comment]
Thanks for reviewing :)

**Pull request merged**